t/test01.t

```diff
@@ -383,6 +387,11 @@ if ( $ENV{ZONEMASTER_RECORD} ) {
 
 done_testing();
 
+# cleaning up
+$dbh->do( "DROP table test_results" );
+$dbh->do( "DROP table users" );
+$dbh->do( "DROP table batch_jobs" );
```

Comment:
Why not drop the tables (IF EXISTS) right before they're created? That way you can study the state of the database after the test is run, in case you need to. And when the test dies in the middle of a run you don't have to drop the dirty tables manually before your next run.